.github/workflows/publish.yml

```diff
@@ -4,7 +4,7 @@ name: Publish
 
 on:
   push:
-    branches: [ main, release-* ]
+    branches: [ main ]
```

Comment:
Do we want to create the bundles for the `stable` channel as well?

Reply:
Was curious about stable branch too

Reply:
Yes, good catch. And to that end, I should throw the branch name in the image tag so we can distinguish between main and stable. I'll make this update.